### ISSUE TITLE
s1~s4 training search setting

### ISSUE THREAD

**Issue by aaron851113** [xiangning-chen/SmoothDARTS#5]
Hi, thanks for your release code.
I ran the experiments about searching/evaluating on s1~s4 on Cifar10, but they got imperfect results as paper's.
Can you share more about hyperparameter setting on these? Thanks!

**Comment by xiangning-chen** [comment]
Hi,

Thanks for your question.
"Results in Table 4 are obtained by running every method 4 independent times and pick the final architecture
based on the validation accuracy (retrain from scratch for a few epochs)." Which is the same as [1].

1. Zela, et al. Understanding and Robustifying Differentiable Architecture Search. ICLR 2020.

**Comment by aaron851113** [comment]
Thanks for your reply! I will try it.
